app/Resources/views/deletionCounter/result.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends 'base.html.twig' %}
+{% import 'macros/layout.html.twig' as layout %}
+{% import 'macros/wiki.html.twig' as wiki %}
+{% import 'macros/pieChart.html.twig' as chart %}
+
+{% block body %}
+<div class="panel panel-primary">
+    {{ layout.userHeading(user, project, xtPage) }}
+
+    <div class="panel-body xt-panel-body">
+        {{ wiki.userLinks(user, project, xtPage) }}
+
+        {% set content %}
+            <div class="col-lg-6">
+            <table class="table table-bordered table-hover table-striped xt-show-hide--target">
```

Comment:
`xt-show-hide--target` is for adding a [show]/[hide] links for subsections within a `panel`. Here we have just the one so this class can be removed. I still need to document this, sorry about that :)